build_component.py

```diff
@@ -295,8 +295,14 @@ def main():
   os.chdir(os.path.join(repos_root_dir, ".."))
   # Copy the necessary files to the respective target folders, 
   # following the instructions in scripts/config_build.txt.
-  config_file = open("scripts/config_build.txt")
-  
+  try:
+    config_file = open("scripts/config_build.txt")
+  except IOError:
+    print "unable to open file scripts/config_build.txt ! This \
+    script is not meant to be called individually, but through \
+    a wrapper script, build.py. See the Seattle build instructions \
+    wiki for details: https://seattle.poly.edu/wiki/BuildInstructions"
+    return 0
```

Comment:
You might want to raise an exception here, or call `sys.exit(1)` (similarly to `help_exit()`'s [approach](https://github.com/SeattleTestbed/common/blob/karthikgvss-patch-1/build_component.py#L221-L223).